### DISCUSSION TITLE
Date Time Type in Rune

### DISCUSSION THREAD

**Discussion by kukut** [rune-rs/rune#660] [Q&A]
How to handle Data Time Type in Rune?
Any module in Rune that can handle that?

**Comment by udoprog** [comment]
There is a very sparse module available in [`rune-modules`](https://github.com/rune-rs/rune/blob/main/crates/rune-modules/src/time.rs).

For more time management you'll have to build your own module, by for example wrapping up [`chrono`](https://docs.rs/chrono).
